site/elements/civievents.php

```diff
@@ -53,16 +53,13 @@ protected function getInput() {
     $config = CRM_Core_Config::singleton();
 
     $params = array(
-      'version' => '3',
       'is_active' => 1,
-      'return.title' => 1,
-      'return.id' => 1,
-      'return.end_date' => 1,
-      'return.start_date' => 1,
-      'rowCount' => 100,
+      'return' => array("title"),
+      'start_date' => array('>=' => "today"),
```

Comment:
should this be <=

Reply:
No, we're looking for future events so where the start date is today or later.